src/util/HeaderUtil.ts

```diff
@@ -99,6 +99,15 @@ export interface AcceptLanguage extends AcceptHeader { }
  */
 export interface AcceptDatetime extends AcceptHeader { }
 
+/**
+ * Contents of a HTTP Content-Type Header.
+ * Optional parameters Record is included.
+ */
+export interface ContentType {
+  value: string;
+  parameters?: Record<string, string>;
```

Comment:
Would not make this optional. Just have an empty object if there are no parameters.